### ISSUE TITLE
Some questions about custom model loading

### ISSUE THREAD

**Issue by ChangtiWu** [eth-sri/language-model-arithmetic#12]
Does it only accept loading open source models from Huggingface? 
What if I want to use the LLM api provided by other platforms, such as the api provided by SiliconFlow, is this ok in such a black-box LLM scenario?

**Comment by JasperDekoninck** [comment]
Hi! Unfortunately, Model Arithmetic requires access to the logits of models and can therefore not be used in a fully black-box setting. While some APIs do support sending logits of the (top k) tokens, this repository does not support this option and there is currently no plan to support this in the future.

**Comment by ChangtiWu** [comment]
Thank you! it's a nice work!
